client/state/notices/reducer.js

```diff
@@ -31,6 +32,16 @@ export const items = createReducer( {}, {
 
 		return omit( state, noticeId );
 	},
+	[ NOTICE_CLEAR ]: ( state ) => {
+		return reduce( state, ( memo, notice, noticeId ) => {
```

Comment:
The problem is, these notices are created in the middleware and discarded from the form component, how can we know in the form component the ids of those notices?